ZUGFeRD/InvoiceDescriptor21Writer.cs

```diff
@@ -572,235 +572,251 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream)
             #endregion
 
             #region ApplicableHeaderTradeSettlement
-            Writer.WriteStartElement("ram:ApplicableHeaderTradeSettlement");
```

Comment:
There are 364 changes to the InvoiceDescriptor21Writer.cs. Why is that?

Reply:
I changed the indentation of the ApplicableHeaderTradeSettlement block for better reading. And this is seen as a change in each line. But everything I have done is added a tab for each line in the block/region